### PR TITLE
[WIP][#12] Initial StatusItem

### PR DESCRIPTION
**Pull request by FalahShazib**
There are some syntax differences when importing since I'm working on a windows environment.

### CODE REVIEW

File: components/StatusItem/index.js
Comment (katrinacrisostomo) [inline components/StatusItem/index.js]: just upload the two pictures into the project in a photos folder and call that url

File: components/StatusItem/index.js
Comment (katrinacrisostomo) [inline components/StatusItem/index.js]: trailing comma

Comment (katrinacrisostomo) [reply components/StatusItem/index.js]: also let's put `propTypes` at the top to keep consistent with George's component

File: components/StatusItem/index.js
Comment (katrinacrisostomo) [inline components/StatusItem/index.js]: code style nit: don't leave spaces around the `=`

File: components/StatusItem/index.js
Comment (katrinacrisostomo) [inline components/StatusItem/index.js]: space after `()`

File: components/StatusItem/index.js
Comment (katrinacrisostomo) [inline components/StatusItem/index.js]: space after `React.Component`

File: components/StatusItem/index.js
Comment (katrinacrisostomo) [inline components/StatusItem/index.js]: space before `/>`

File: components/StatusItem/index.js
Comment (katrinacrisostomo) [inline components/StatusItem/index.js]: trailing comma

File: components/StatusItem/index.js
Comment (katrinacrisostomo) [inline components/StatusItem/index.js]: Let's not call the same thing both `danceNum` and `number` in different places, let's just call the prop `danceNum` so that it's (1) more specific, (2) the same as the style name

**Comment by FalahShazib** [comment]
I've made the specified changes. Additionally, I've attached a screenshot of sample items. 

screenshot_20180306-083211

**Comment by FalahShazib** [comment]
New format
screenshot_20180306-232935

File: components/StatusItem/index.js
Comment (katrinacrisostomo) [inline components/StatusItem/index.js]: this block is indented too much

**Comment by katrinacrisostomo** [comment]
@FalahShazib I'm going to install `jslint` into our repo first before we merge this in so that you can use that to change around your spacing and stuff to follow a unified javascript style with the rest of our repo, so let's not merge this until after that 👍 

bitmoji

File: components/StatusItem/index.js
Comment (katrinacrisostomo) [inline components/StatusItem/index.js]: line 10 - line 13 is not indented enough

Comment (katrinacrisostomo) [reply components/StatusItem/index.js]: seems like you're just setting a value on `pic` without first declaring it with `var`/`let`/`const` etc. which isn't best code practice.  pls do that first :) 

lmk if you have any questions!

File: components/StatusItem/index.js
Comment (katrinacrisostomo) [inline components/StatusItem/index.js]: I belieeeve you don't want spaces here surrounding the string you want rendered in this `Text` block. 

For example:
what you're doing is this:
```
<Text> hello world! </Text>   
```
this will render Text ' hello world! ', string with surrounding spaces making spacing like margins on it look jenky

what we want is this:
```
<Text>hello world!</Text>    
```
this will render Text 'hello world!', correct string with no surrounding spaces

File: components/StatusItem/index.test.js
Comment (katrinacrisostomo) [inline components/StatusItem/index.test.js]: don't forget to make these test descriptions more specific so that they're all different from one another! they should all have a different test description

right now, if one of these tests was failing it would just tell us `renders without crashing` from `components/StatusItem/index.test.js` is failing but we wouldn't know which of these tests it is 😆

File: components/StatusItem/index.js
Comment (katrinacrisostomo) [inline components/StatusItem/index.js]: Let's specify the `propTypes` at the top of this `Information` component too!

**Comment by FalahShazib** [comment]
New Look + Using SVG Icons
screenshot_20180311-190601

File: components/StatusItem/index.js
Comment (georgeke) [inline components/StatusItem/index.js]: Instead of hardcoding each string for each case, can you create a constant in this file:

```
const DANCE_STATUS = {
  uploaded: 0,
  loading: 1,
  requiredConnection: 2,
}
```

Which can then be exported and imported when we want to use it (e.g. index.test.js below)

File: components/StatusItem/index.js
Comment (georgeke) [inline components/StatusItem/index.js]: I don't think you need to have a separate `Information` and `StatusItemIcon` component since they are only used in this file. It'll be cleaner to just combine everything into `StatusItem` instead of having to pass the same props around.

**Review by georgeke — CHANGES_REQUESTED** [review]
Can you make styling match jeremy's design? (i.e. black background, no gray border for each status item,  and white text).

**Comment by FalahShazib** [comment]
New look 

screenshot_20180313-225041

File: components/StatusItem/index.js
Comment (katrinacrisostomo) [reply components/StatusItem/index.js]: yo tru

**Review by katrinacrisostomo — APPROVED** [review]
LGTM!

**Review by georgeke — APPROVED** [review]

**Pull request merged**
